caffe2/operators/experimental/c10/schemas/fc.h

```diff
@@ -12,6 +12,7 @@ struct FullyConnected final {
 
   struct Cache final {
     vector<int64_t> Y_shape_cache_;
+    // TODO: Remove Tensor{CPU}
```

Comment:
This bias_multiplier_ tensor is only used in caffe2/operators/experimental/c10/cpu/fc_cpu.cpp. That's where your Reinitialize call should be.

Reply:
But now it's a C10Tensor isn't it? do we have to write a ReinitializeTensor for C10Tensor as well?

Reply:
C10Tensor is just a thin wrapper around TensorImpl. It doesn't do anything different internally. If the kernel expects the tensor to be initialized the caffe2 way (which fc probably does), it has to be initialized the caffe2 way.